app/src/test/java/com/kickstarter/viewmodels/NativeCheckoutRewardViewHolderViewModelTest.kt

```diff
@@ -394,36 +392,9 @@ class NativeCheckoutRewardViewHolderViewModelTest : KSRobolectricTestCase() {
         this.showPledgeFragment.assertNoValues()
 
         // When a reward from a live project is clicked, start checkout.
-        this.vm.inputs.rewardClicked()
+        this.vm.inputs.rewardClicked(2)
         this.showPledgeFragment.assertValue(Pair.create(liveProject, reward))
-    }
-
-    @Test
-    fun testGoToViewPledge() {
-        val liveProject = ProjectFactory.backedProject()
-        val successfulProject = ProjectFactory.backedProject().toBuilder()
-                .state(Project.STATE_SUCCESSFUL)
-                .build()
-
-        setUpEnvironment(environment())
-
-        this.vm.inputs.projectAndReward(liveProject, liveProject.backing()?.reward() as Reward)
-        this.startBackingActivity.assertNoValues()
-
-        // When the project is still live, don't go to 'view pledge'. Should go to checkout instead.
-        this.vm.inputs.rewardClicked()
-        this.startBackingActivity.assertNoValues()
-
-        // When project is successful but not backed, don't go to view pledge.
-        this.vm.inputs.projectAndReward(successfulProject, RewardFactory.reward())
-        this.vm.inputs.rewardClicked()
-        this.startBackingActivity.assertNoValues()
-
-        // When project is successful and backed, go to view pledge.
-        this.vm.inputs.projectAndReward(successfulProject, successfulProject.backing()?.reward() as Reward)
```

Comment:
are we able to delete most of these tests because we've removed the `startBackingActivity` observable?

Reply:
yeah, the `BackingActivity` is part of the old Checkout flow. looks like this 
<img src=https://user-images.githubusercontent.com/1289295/66167776-20268400-e609-11e9-9b64-b216649b530b.png width=330/>

but now we have it in the view/manage screen like this (staging gives fake payment data):
<img src=https://user-images.githubusercontent.com/1289295/66167811-43513380-e609-11e9-97e9-08391be3509d.png width=330/>

Reply:
gonna update that description to note that!